404.php

```diff
@@ -6,11 +6,286 @@
+  },
+
+  preparePoints: function() {
+
+    // Clear the current points
+    this.points = [];
+
+    var width, height, i, j;
+
+    var colors = this.bgContextPixelData.data;
+
+    for( i = 0; i < this.canvas.height; i += this.density ) {
+
+      for ( j = 0; j < this.canvas.width; j += this.density ) {
+
+        var pixelPosition = ( j + i * this.bgContextPixelData.width ) * 4;
+
+        // Dont use whiteish pixels
+        if ( colors[pixelPosition] > 200 && (colors[pixelPosition + 1]) > 200 && (colors[pixelPosition + 2]) > 200 || colors[pixelPosition + 3] === 0 ) {
+          continue;
+        }
+
+        var color = 'rgba(' + colors[pixelPosition] + ',' + colors[pixelPosition + 1] + ',' + colors[pixelPosition + 2] + ',' + '1)';
+        this.points.push( { x: j, y: i, originalX: j, originalY: i, color: color } );
+
+      }
+    }
+  },
+
+  updatePoints: function() {
+
+    var i, currentPoint, theta, distance;
+
+    for (i = 0; i < this.points.length; i++ ){
+
+      currentPoint = this.points[i];
+
+      theta = Math.atan2( currentPoint.y - this.mouse.y, currentPoint.x - this.mouse.x);
+
+      if ( this.mouse.down ) {
+        distance = this.reactionSensitivity * 400 / Math.sqrt((this.mouse.x - currentPoint.x) * (this.mouse.x - currentPoint.x) +
+         (this.mouse.y - currentPoint.y) * (this.mouse.y - currentPoint.y));
+      } else {
+        distance = this.reactionSensitivity * 200 / Math.sqrt((this.mouse.x - currentPoint.x) * (this.mouse.x - currentPoint.x) +
+         (this.mouse.y - currentPoint.y) * (this.mouse.y - currentPoint.y));
```

Comment:
i'd consider using more variables and `Math.pow( ____, 2)` to reduce code duplication and make it a bit more readable.
```js
let diffX = this.mouse.x - currentPoint.x;
let diffY = this.mouse.y - currentPoint.y;

let sensitivityModifier = (this.mouse.down) ? 400 : 200; 
let distance = (this.reactionSensitivity * sensitivityModifier) / Math.sqrt(  Math.pow(diffX, 2) + Math.pow(diffY, 2) );
```